cf-deployment.yml

```diff
@@ -1426,27 +1436,16 @@ instance_groups:
         cert: "((loggr_udp_forwarder_tls.certificate))"
         key: "((loggr_udp_forwarder_tls.private_key))"
         server_name: loggr_udp_forwarder_metrics
-- name: doppler
+- name: log-cache
   azs:
   - z1
   - z2
-  instances: 4
-  vm_type: minimal
+  instances: 1
```

Comment:
Why 1 instance instead of more?

Reply:
Doppler is set to 4.

Reply:
If we do change this instance number, we should probably add a line to the bosh-lite ops file to put the instance count back to 1.

Reply:
We couldn't find a reason to have more than one (log cache does not replicate data, so having two would just add more memory, not reliability) and wanted to keep a minimal footprint.